### PR TITLE
make deepdiff optional

### PR DESCRIPTION
**Pull request by dlwh**

### CODE REVIEW

File: src/levanter/store/cache.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from dataclasses import dataclass
 from typing import Any, Callable, Dict, Iterator, List, Optional, Sequence, TypeVar, Union
 
-import deepdiff
+# import deepdiff
```
Comment (percyliang) [inline src/levanter/store/cache.py:17]: Remove comment?

**Review by percyliang — COMMENTED** [review]
What's the motivation for making this optional?  Can we not include it like a normal package?

**Comment by dlwh** [comment]
so we can avoid a full redeploy of the marin cluster

**Comment by percyliang** [comment]
What's the problem with a full redeploy?  It seems like we want to be able to redeploy easily anyway...

**Comment by dlwh** [comment]
i dunno, i'll close this.

**Pull request closed without merging**